$OpenBSD: patch-make_hotspot_lib_CompileJvm_gmk,v 1.2 2021/10/30 22:48:11 kurt Exp $

threadLS_bsd_aarch64.S name changed in 17 and needs to be excluded.
Disable unused-but-set-parameter for llvm13.

Index: make/hotspot/lib/CompileJvm.gmk
--- make/hotspot/lib/CompileJvm.gmk.orig
+++ make/hotspot/lib/CompileJvm.gmk
@@ -95,7 +95,8 @@ DISABLED_WARNINGS_clang := tautological-compare \
     undefined-var-template sometimes-uninitialized unknown-pragmas \
     delete-non-virtual-dtor missing-braces char-subscripts \
     ignored-qualifiers missing-field-initializers mismatched-tags \
-    unnamed-type-template-args shift-negative-value format-nonliteral
+    unnamed-type-template-args shift-negative-value format-nonliteral \
+    unused-but-set-parameter
 
 ifneq (,$(filter 1%,$(CC_VERSION_NUMBER)))
   DISABLED_WARNINGS_clang += misleading-indentation
@@ -128,7 +129,7 @@ else ifeq ($(call And, $(call isTargetOs, linux) $(cal
 endif
 
 ifeq ($(OPENJDK_TARGET_OS_ENV)-$(OPENJDK_TARGET_CPU), bsd.openbsd-aarch64)
-  JVM_EXCLUDE_FILES += threadLS_bsd_aarch64.s
+  JVM_EXCLUDE_FILES += threadLS_bsd_aarch64.S
 endif
 
 ifeq ($(call isTargetOs, linux macosx windows bsd), true)
